daemon/cmd/daemon_main.go

```diff
@@ -1918,6 +1919,45 @@ func startDaemon(d *Daemon, restoredEndpoints *endpointRestoreState, cleaner *da
 	if err != nil {
 		log.WithError(err).Error("Unable to store Viper's configuration")
 	}
+
+	checkEndpointBPFProgs(d)
+}
+
+// checkEndpointBPFProgs check to ensure endpoints associated BPF programs remain loaded.
+func checkEndpointBPFProgs(d *Daemon) {
+	const (
+		epsBPFSync         = "eps-bpf-sync"
+		epsBPFSyncDuration = 30 * time.Second
```

Comment:
Should we consider making this tunable and possible to disable it?

Reply:
I think having a configurable interval makes sense. However not sure about disabling it? Should resiliency be optional?